mapping/requirements_officejs.json

```diff
@@ -645,7 +645,7 @@
 							"apiVersion": "1.1",
 							"supportedProductVersions": [{
 								"from": {
-									"build": "15.20",
+									"build": "15.20.203.0",
```

Comment:
Are you sure the build number is not 15.**0**.203.0?
For the 16.* versions I updated earlier, the minor version was 0 for all the four digit build numbers. like 16.**0**.x.y

Reply:
Updated